packages/web3-eth-personal/package.json

```diff
@@ -22,7 +22,7 @@
 		"test:ci": "jest --coverage=true --coverage-reporters=json --verbose",
 		"test:watch": "npm test -- --watch",
 		"test:unit": "jest --config=./test/unit/jest.config.js",
-		"test:integration": "jest --config=./test/integration/jest.config.js --runInBand --passWithNoTests"
+		"test:integration": "jest --config=./test/integration/jest.config.js --forceExit --runInBand"
```

Comment:
I could totally be wrong, but it seems like the use of `--forceExit` is hiding some bugs and @nazarhussain had previously mentioned

Reply:
Without `--forceExit` it looks like the personal [testcases](https://github.com/ChainSafe/web3.js/runs/6457629918?check_suite_focus=true) pass, but runs forever as the ws it never disconnects.
I would need to introduce the providers ws package in my test suite and disconnect if i were to take away forceExit.